tool/mediapipe_ci_script_master.sh

```diff
@@ -8,7 +8,9 @@ source "$DIR/ci_script_shared.sh"
 flutter doctor -v
 
 declare -ar PACKAGE_NAMES=(
+    # TODO(craiglabenz): Uncomment once native assets works on CI
```

Comment:
What is the state of this? Is there a tracking issue?

Reply:
I pulled the ole _complete the TODO without removing the TODO_ trick